app/assets/stylesheets/user.scss

```diff
@@ -0,0 +1,62 @@
+@import "application";
+
+.sign-up-container {
+  border: 2px;
+  font-family: 'Open Sans', sans-serif;
+  position: relative;
+  top: 8em;
+  width: 30% !important;
+
+}
+
+.sign-up-header {
+  background-color: $my-brown;
+  border-radius: 8px 8px 0px 0px;
+  color: $my-white;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1.0em;
+  font-weight: lighter;
+  height: 7em;
+  opacity: 0.95;
+  padding: 1em;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+}
+
+.sign-up-box {
+  background-color: rgba(56,65,69,0.4);
+  border-radius: 0px 0px 8px 8px;
+  padding: 2em;
+}
+
+.label-font {
+  color: $my-white;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 0.9em;
+  a {
+    color: $my-black;
+    font-weight: bold;
+    text-decoration: none;
+  }
+}
+
+.username-field {
+  border-radius: 0.2em !important;
+  color: $my-black !important;
+  height: 1.0em !important;
```

Comment:
!important should not be used<br>`1.0em` should be written without the mantissa as `1em`